pages/adventure/adventure.js

```diff
@@ -0,0 +1,100 @@
+    name: "Innkeeper",
+    dialog: [
+      `Vaya vaya, así que este es el nuevo trabajador. ¡Bienvenido, muchacho! Espero que disfrutes de este espacio durante tu estadía trabajando.`,
+      `Hahaha, ¿sorprendido? ¿Acaso esperabas otra imagen de mí? Eso es lo que dicen todos al conocerme, los rumores me pintan de una forma muy diferente, ¿verdad? Por suerte eso a mí no me molesta.
+         Sería terrible si me molestase lo que el resto dice de mí, ¿no te parece, muchacho?`,
+      `Me comentaron que llegaste aquí por una oferta que pusimos en el Gremio, supongo que serás aventurero. La realidad es que no solemos contratar aventureros pero se trata de una rara excepción...`,
+      `Esta semana tenemos un evento importante y necesitamos de gente que... sea hábil. Tenemos muchas influencias de la que hacerse cargo y mucha mercancía valiosa que cuidar. Y espero que nos puedas ayudar.`,
+      `Nuevamente te doy la bienvenida a "Hotel Del Luna" y espero no tengamos ningún problema con vos también, ¿está bien, muchacho?`,
+      `¡Que disfrutes de tu estadía en este maravilloso y mágico hotel!`,
+    ],
+    background: "/images/89699065dfcc78b45a4e2b79817a7219.png",
+    foreground: "/images/Gati-blanca.png",
+  },
+  {
+    id: "2",
+    name: null,
+    dialog: [
+      `(No sé qué habrá sido esa última advertencia pero tendré que tener cuidado si no quiero tener problemas aquí.)`,
+      `(Camino hacia mi habitación absorto en mis pensamientos, ordeno mi equipaje después de un largo viaje hasta el hotel y cuando dispongo a relajarme en la cama escucho un sonido extraño en la habitación.)`,
+      `(Me acerco al lugar de donde provenía el ruido y me encuentro con una sorpresa...)`,
+    ],
+    background:
+      "/images/RenderInteriorConcept15-1920x1080-be448c2077a2576eb46d68d4d4100323.jpg",
+    foreground: null,
+  },
+  {
+    id: "3",
+    name: "Mysterious Character",
+    dialog: [`Zzz...`, `...`],
+    background:
+      "/images/RenderInteriorConcept15-1920x1080-be448c2077a2576eb46d68d4d4100323.jpg",
+    foreground: "/images/Gato-colorado-atigrado.png",
+  },
+];
+
+const backgroundImageDom = document.getElementById("background");
+const foregroundImageDom = document.getElementById("foreground");
+const characterNameDom = document.getElementById("name");
+const dialogTextsDom = document.getElementById("dialog");
+let currentDialogIndex = 0;
+let selectedScene;
+let currentScene = 0;
+
+function populateDom(backgroundImage, foregroundImage, characterName, dialogTexts) { //populate Dom Elements
+  backgroundImageDom.setAttribute("src", `${backgroundImage}`);
+  if (foregroundImage != null) {
+    if (foregroundImageDom.classList.contains("hidden") == true) { //si existe una clase CSS que oculta el elemento y existe foreground, quita la clase CSS
+      foregroundImageDom.classList.remove("hidden");
+    }
+    foregroundImageDom.setAttribute("src", `${foregroundImage}`); //cambia el src del elemento (cambia el foreground)
```

Comment:
La interpolacion esta de mas.

```suggestion
    foregroundImageDom.setAttribute("src", foregroundImage); //cambia el src del elemento (cambia el foreground)
```